viz/charts/linkerd-viz/templates/prometheus.yaml

```diff
@@ -70,7 +73,7 @@ data:
       kubernetes_sd_configs:
       - role: pod
         namespaces:
-          names: ['{{.Values.global.namespace}}']
+          names: ['{{.Values.linkerdNamespace}}']
       relabel_configs:
       - source_labels:
         - __meta_kubernetes_pod_label_linkerd_io_control_plane_component
```

Comment:
Note that this scrape config wil now just scrape the core components. We have a separate section for Grafana. Worth adding separate sections for Viz and Jaeger?

Reply:
Hmm, for grafana we already had specific metrics in mind that we felt users would find useful but I don't think we have that kind of knowledge yet with Jaeger. We can definitely replicate the same for other linkerd components in viz.

Reply:
I'm a bit against on having these jobs for each induvidual extension/component, etc. Instead can we have a job that can cover all linkerd components irrespective of the extension. Something like

```
    - job_name: 'linkerd-components'
      kubernetes_sd_configs:
      - role: pod
      relabel_configs:
      - source_labels:
        - __meta_kubernetes_pod_label_linkerd_io_extension
        - __meta_kubernetes_pod_label_linkerd_io_control_plane_component
        - __meta_kubernetes_pod_container_port_name
        action: keep
        regex: (.*);(.*);admin-http$
      - source_labels: [__meta_kubernetes_pod_container_name]
        action: replace
        target_label: component
```

Reply:
Ok I've created #5429 to track that effort in a separate PR, alongside with the required change in the Grafana chart.